gulpfile.js

```diff
@@ -64,16 +64,44 @@ gulp.task('instrument', function () {
     .pipe(istanbul.hookRequire());
 });
 
+function gulp_test_app() {
+    return gulp.src([
+    ])
+    .pipe(mocha({
+        log: true,
+        timeout: 5000,
+        reporter: 'spec',
+        ui: 'bdd',
+        ignoreLeaks: true,
+        globals: ['should']
+    }));
+}
+
 function gulp_test() {
-    return gulp.src('test/**/*.spec.js')
-        .pipe(mocha({
-            log: true,
-            timeout: 5000,
-            reporter: 'spec',
-            ui: 'bdd',
-            ignoreLeaks: true,
-            globals: ['should']
-        }));
+    var argv = require('yargs').argv;
```

Comment:
I can easily convert to use minimist, just happened to find an easy to copy and paste example with yargs.